daemon/src/wire.rs

```diff
@@ -174,6 +175,10 @@ pub enum MakerToTaker {
         oracle_event_id: BitMexPriceEventId,
         tx_fee_rate: TxFeeRate,
         funding_rate: FundingRate,
+        /// Fee to be paid for the rollover.
+        ///
+        /// The maker comes up with this amount so that both parties are on the same page
```

Comment:
🔧 Probably better if this docstring goes on the type.

Reply:
Moved in [`75af806` (#1981)](https://github.com/itchysats/itchysats/pull/1981/commits/75af8063283ec90b15367747f2601f8dc756f1e0)